docs/env-providers.md

```diff
@@ -28,6 +28,7 @@ Usage:
 ```bash
 export KUBEVIRT_PROVIDER=k8s-1.10.4 # choose this provider
 export KUBEVIRT_NUM_NODES=3 # master + two nodes
+export KUBEVIRT_MEMORY_SIZE=4096
```

Comment:
Why do I have to extra export the default value for every provider?

Reply:
You don't have to export default value. It is for user to show him, he can change the memory size.